### PR TITLE
Sion Bell HW01

### PR DESCRIPTION
**Pull request by sionbell**
I wanted to use numerous header sizes, so I put two hashtags before my name to make it a header 2 size and four hashtags before "Introduction" to make it a header 4 size. To italicize the sources of the data I am using for my thesis, I surrounded the names of the studies by asterisks. To make the components of my thesis a list, I put a single asterisk and a space before the names of the policies. Originally, I did not include an empty line after the list, so the text for the list flowed into the rest of the sentence rather than there being a clear break between the list and the rest of the text. I fixed this by adding an empty line after the last part of the list. To embed a photo of myself, I initially copied the path to an image I uploaded to my repository on github and followed the code guidelines on https://www.rstudio.com/wp-content/uploads/2015/03/rmarkdown-reference.pdf. It took a few tries to realize that I needed to upload the image to my "hw01" repository rather than my "myrepo" repository. I resized the image, uploaded it to my "hw01" repo, copied the new file path, and replaced the original path with the new one.

### CODE REVIEW

**Comment by cnevakivicallanan** [comment]
Hi Sion,

Here's your peer evaluation! (From me, Cleo Nevakivi-Callanan.)

Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            | x          |              |            |
| **Coding strategy**         | x          |              |            |
| **Presentation**            | x          |              |            |
| **Achievement, creativity** | x          |              |            |
| **Ease of access**          | x          |              |            |

Remarks:

* Your project includes bold/italicized text, headings, bullet points, and photographs, but it does look like the link portion is missing! If you were to repeat this project, I would recommend maybe adding a link to the NES/GSS to fill that requirement.
* I appreciate the specificity in your research interests, especially after seeing your public opinion project from our other class. I'd be interested to hear more about your thesis findings.
* I did not know that you could upload images to GitHub and make photos work that way— I'll have to try that next time.
* It might make your bullet pointed list a little bit clearer if you included the final element (which, currently, stretches into a new paragraph) into its own bullet point. The current format is a little bit confusing.
* Perhaps you might find this [R Markdown cheatsheet](https://www.rstudio.com/wp-content/uploads/2015/03/rmarkdown-reference.pdf) as helpful as I did.

Great work!

File: README.md [hunk 1]
```diff
@@ -1,2 +1,20 @@
-# hw01
-Repository for homework 01
+# hw01
```
Comment (jfan3) [inline README.md:1]: Redundant here.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,20 @@
-# hw01
-Repository for homework 01
+# hw01
+Repository for homework 01
+
+## Sion Bell
+
+#### Introduction
+
+![Photo of me in the UK from this summer](DSC_0273%20(2).JPG)
```
Comment (jfan3) [inline README.md:8]: Nice!

File: README.md [hunk 3]
```diff
@@ -1,2 +1,20 @@
...
+
+My name is Sion, and I am currently a MAPSS student in Political Science, specifically in American politics. I am interested in public opinion on socioeconomic issues, particularly policies like 
+
+* Medicare
+* Social Security
+* Income tax rates
+* Minimum wage rates
+* Other aspects of the social safety net
+
+and political beliefs on broad questions like how large or small the government should be. For my thesis, I am employing data from the *American National Election Studies* and the *General Social Survey*.
```
Comment (jfan3) [inline README.md:18]: The "and" here could be another bullet point?

File: README.md [hunk 4]
```diff
@@ -1,2 +1,20 @@
...
+
+As an undergraduate, I majored in Anthropology and African American Studies at the University of Kansas.
```
Comment (jfan3) [inline README.md:20]: It would be nice to create sections with `###`

File: README.md [hunk 5]
```diff
@@ -1,2 +1,20 @@
-# hw01
-Repository for homework 01
+# hw01
+Repository for homework 01
+
+## Sion Bell
+
+#### Introduction
```
Comment (jfan3) [inline README.md:6]: Why four `#`? Maybe it would be nice to have `###` and multiple sections.

**Review by jfan3 — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |             |x              |            |
| **Coding strategy**         |           |x              |            |
| **Presentation**            |           |x              |            |
| **Achievement, creativity** |           |              |x            |
| **Ease of access**          |x           |              |            |

Remarks:

* It would be nice if you can explore adding links. 
* It would also be nice if you can try playing with fonts (bold, underscore, etc).

**Comment by jongriffiths** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |   x        |              |            |
| **Coding strategy**         |     x      |              |            |
| **Presentation**            |     x      |              |            |
| **Achievement, creativity** |    x       |              |            |
| **Ease of access**          |      x     |              |            |

Remarks: 
I don't have much to add, but I think you did a great job! I also liked the picture. Addtionally, you did everything succinctly and efficiently which made it easy to review. All the code and formatting was very intuitive. I also used the markdown guide you to which you linked. I found it really helpful, too.

**Comment by jessrobinson42** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      x     |              |            |
| **Coding strategy**         |     x      |              |            |
| **Presentation**            |           |      x       |            |
| **Achievement, creativity** |           |     x         |            |
| **Ease of access**          |      x     |              |            |

Remarks:

* Elaborate on above, especially for "needs work."
Overall your page seems to meet all the requirements. It uses lists, photos, and section breaks, and is easy to navigate. You could potentially add more about yourself or external links for more creativity, but overall I though your presentation was concise and interesting. 

* Some specific praise?
I liked that you chose to you upload an image to your repository. I just used a link for mine, so I admire the work you put into that. 

* Something I learned?
I learned about a good resource for uploading files into Git Hub, as well as about cool American Politics datasets. 

* Specific constructive criticism?
I think you could potentially make your page look cleaner by deleted the HW01 repository from the top. Similarly, I think making sure you last bullet point doesn't start into a new paragraph would make your page a little easier to read. 

* Something I know and that you, my peer, might like to know because it is relevant to something you struggled with.
I think changing where you put the lines breaks in the last point in the list / beginning of the next line might help with the above.

**Pull request closed without merging**